packages/arb-token-bridge-ui/src/components/TransferPanel/MoveFundsButton.tsx

```diff
@@ -0,0 +1,49 @@
+import { twMerge } from 'tailwind-merge'
+
+import { useNetworks } from '../../hooks/useNetworks'
+import { getBridgeUiConfigForChain } from '../../util/bridgeUiConfig'
+import { useAppContextState } from '../App/AppContext'
+import { Button } from '../common/Button'
+import { useTransferReadiness } from './useTransferReadiness'
+import { useAccountType } from '../../hooks/useAccountType'
+import { useNetworksRelationship } from '../../hooks/useNetworksRelationship'
+import { getNetworkName } from '../../util/networks'
+
+export function MoveFundsButton({
+  onClick
+}: React.ButtonHTMLAttributes<HTMLButtonElement>) {
```

Comment:
Can we either spread props to in the button props or change the type to `Pick<React.ButtonHTMLAttributes<HTMLButtonElement>, 'onClick'>`

Otherwise we can pass props to the component but they do nothing